desktop/components/react/utils/renderReactLayout.js

```diff
@@ -116,8 +121,7 @@ export function renderReactLayout (options) {
         )
       }
     }
-
-    return html
+    return { html, styledComponentCSS }
```

Comment:
Minor nit: I would just name this `css` rather than leaking an implementation detail like us using styled-components.